doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc

```diff
@@ -3,191 +3,54 @@
 
 This section highlights new features in {PRODUCT} {PRODUCT_VERSION}.
 
-ifdef::DM[]
+== Ability to only enable decision management components and functionality
 
-== {PRODUCT} {ENTERPRISE_VERSION} Kogito decision services support
+You can now configure the deployment to only enable decision management related components and functionality so that jBPM and case management features are disabled.
 
-=== IBM Power support in {OPENSHIFT} 4.10
+== Ability to use a custom registry and specific images
 
-Starting with {PRODUCT} {ENTERPRISE_VERSION}, IBM Power supports {OPENSHIFT} 4.10 using Open JDK 11. For a complete list of supported configurations, see https://access.redhat.com/articles/3354301[Red Hat Decision Manager 7 Supported Configurations].
+You can now configure the custom registry to use specific images during the configuration of the environment.
 
-endif::DM[]
+== Ability to output error logs
 
-== Ability to send cookies in the REST work item handler
+You can now set the `org.kie.jbpm.event.emitters.eagerInit` property to `true` to output the error log when {KIE_SERVER] starts and initializes the Kafka emitter.
 
-You can now send cookies to the REST work item handler using the `Cookie` header and cookie parameters in a REST API as shown in the following example:
+== Support for Quarkiverse File Vault and keystore files
 
-`cookieParam1=cookieParam1_Value;cookieParam2=cookieParam2_Value`
+You can use the Quarkiverse File Vault extension to store credentials as keystore files and use the `file` method to use the keystore files with the Process Instance Migration (PIM) Keystore Vault.
 
-You can also define the path for which the cookie is valid using the `CookiePath` header. By default, the value of `CookiePath` is set to `/`.
+== Ability to use a secret for Admin user during KieApp configuration
```

Comment:
There's a few items that seem to be related to configuring the cloud-based setup.  Would it make sense to bundle them under one title, like we do for DM designer?